server/sonar-db-dao/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -24,7 +24,8 @@
     p.copy_component_uuid as copyComponentUuid,
     p.developer_uuid as developerUuid,
     p.authorization_updated_at as authorizationUpdatedAt,
-    p.created_at as createdAt
+    p.created_at as createdAt,
+    p.tags as tagsString
```

Comment:
it's a personal opinion, but I prefer placing date columns at the end, like you have done just after in the insert